beta/README.md

```diff
@@ -1,5 +1,121 @@
+
+A deployment example may be in beta for whatever length of time is required to meet the promotion criteria. 
+
+- Functionality
+  - The deployment example _must_ work for all identity providers 1Password supports (unless otherwise exempt from that criteria). 
+  - The deployment example _must_ be capable of scaling to accommodate different provisioning needs (unless otherwise exempt from that criteria).
+  - The deployment example _must_ provide the complete automated user provisioning experience described by 1Password and in the documentation accompanying the deployment example. 
+  - Feedback from customers using the deployment example is nearly universally positive. Any critical feedback has been addressed to the greatest possible extent during the preceeding beta period. 
+  - The deployment example _must_ provide the complete automated user provisioning experience described by 1Password and in the documentation accompanying the deployment example. 
+  - All documented configurations _must_ be tested by the author and at least one reviewer. Tests must reliably succeed.
+  - Some number of customers, greater than one (but no specific minimum is prescribed), are known to have used the deployment example in whatever is deemed a "standard" or common configuration for that example.
+    - There must be documentation of these deployments, such as if it was assisted by 1Password Solutions Architect or developer. 
+  - Ideally at least _one_ customer has used the deployment example to deploy 1Password SCIM bridge in other, less typical, configurations, if applicable. 
+    - There must be documentation of these deployments, such as if it was assisted by 1Password Solutions Architect or developer.
+  - SCIM bridges deployed using the deployment example _must_ be in use by customers through at least one SCIM bridge update release with no evidence of failure or issues. 
+- Documentation
+  - Documentation _must_ conform to the [1Password Style Guide](https://support.1password.com/style-guide/).
+  - Documentation _must_ include the following:
+    - Complete instructions for deploying 1Password SCIM Bridge
+      - If there are multiple potential configurations, all configurations must be documented and configuration examples provided. 
+    - Complete instructions for updating the SCIM Bridge.
+    - Complete instructions for scaling the SCIM bridge, if applicable.
+
+### Process for promoting a deployment example out of beta
+
+Once the above criteria have been met, the deployment example can be promoted out of beta in the following way:
+- Create a branch called `promote/<deployment example name>`
+- On that branch, move the directory containing the documentation and configuration manifest from `./beta` to the root of the repository. 
+- Review all documentation and URL/URI paths in any related files to ensure that the 'beta' is removed from the paths
+- File MR to have /scim-update/ to include SCIM bridge update steps for non-beta scim-example. 
```

Comment:
Maybe make this more obvious? File MR to have 1Password Support ...

Reply:
Yowza that really was a half thought, wasn't it.

Reply:
I..... actually think we can exclude this step altogether. We would probably only consider this is the promoted deployment was extremely popular (a la Container Apps). For the majority of cases we can just include update steps in README for the example.

Reply:
I take that back. Let's have it here anyway because we've seen that beta deployments can become the de factor standard quickly.